src/platforms/common/distributed-tracing/instrumentation/custom-instrumentation.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: Custom Instrumentation
+sidebar_order: 40
+description: ""
+supported:
+  - java
+  - javascript
+  - php
+  - python
+---
+
+Distributed tracing works out of the box for the following frameworks: [Django](/platforms/python/guides/django/distributed-tracing/instrumentation/automatic-instrumentation/), [FastAPI](/platforms/python/guides/fastapi/distributed-tracing/instrumentation/automatic-instrumentation/),[Flask](/platforms/python/guides/flask/distributed-tracing/instrumentation/automatic-instrumentation/), [Bottle](/platforms/python/guides/bottle/distributed-tracing/instrumentation/automatic-instrumentation/), [Falcon](/platforms/python/guides/falcon/distributed-tracing/instrumentation/automatic-instrumentation/), [Pyramid](/platforms/python/guides/pyramid/distributed-tracing/instrumentation/automatic-instrumentation/), [Quart](/platforms/python/guides/quart/distributed-tracing/instrumentation/automatic-instrumentation/), [Starlette](/platforms/python/guides/starlette/distributed-tracing/instrumentation/automatic-instrumentation/), [Starlite](/platforms/python/guides/starlite/distributed-tracing/instrumentation/automatic-instrumentation/), [Tornado](/platforms/python/guides/tornado/distributed-tracing/instrumentation/automatic-instrumentation/).
+
+If you are using one of the frameworks listed above, you do not need to do anything for distributed tracing to work! You can now stop reading this page and continue building and amazing product.
+
+If you have a vanilla Python project or use a (yet) unsupported framework you can still add distributed tracing to your project but need to it "by hand".
+
+Two things need to be done for distributed tracing to work:
+
+- When receiving incoming requests, parse and store incoming tracing information.
+- When doing outgoing requests, add tracing information to the outgoing request.
+
+## Parse incoming tracing information
+
+Incoming tracing information has to be parsed and saved.
+Incoming tracing can come from different places:
```

Comment:
"Incoming tracing" is used twice - maybe some other wording can be found?

Reply:
I am always more for using exact the same same term for the same thing. It is not a novel but a documentation that mostly non native speakers read, it should be as easy as possible to follow :-)